jobs/build/ocp/Jenkinsfile

```diff
@@ -401,8 +424,84 @@ node(TARGET_NODE) {
             }
 
             stage( "build images" ) {
-                // TODO: Create a dynamic .repo file pointing to the exact puddle we built instead of "building" so that we can run X.Y builds in parallel
-                sh "ose_images.sh --user ocp-build build_container --branch rhaos-${BUILD_VERSION}-rhel-7 --group base --repo http://download.lab.bos.redhat.com/rcm-guest/puddles/RHAOS/repos/aos-unsigned-building.repo"
+                dir( temp=true ) {
+                  // TODO: Create a dynamic .repo file pointing to the exact puddle we built instead of "building" so that we can run X.Y builds in parallel
+                  PUDDLE_REPO = "http://download.lab.bos.redhat.com/rcm-guest/puddles/RHAOS/repos/aos-unsigned-building.repo"
+                  def build_result = sh (
+                    script: "ose_images.sh --user ocp-build build_container --branch rhaos-${BUILD_VERSION}-rhel-7 --workingdir ${pwd()} --group base --repo ${PUDDLE_REPO}"
+                    returnStatus: true
+                  ) > 0
+                  def failed_list_file = "${pwd()}/logs/failed_image_list"
+                  IMG_FAILURES = []
+                  if(fileExists(failed_list_file)) {
+                    def failed_image_list = readFile(failed_list_file)
+                    IMG_FAILURES = failed_image_list.split("\\r?\\n");
+                  }
+
+                  def success_list_file = "${pwd()}/logs/success_image_list"
+                  IMG_SUCCESSES = []
+                  if(fileExists(success_list_file)){
+                    def success_image_list = readFile(success_list_file)
+                    IMG_SUCCESSES = success_image_list.split("\\r?\\n");
+                  }
+
+                  required_images = [
```

Comment:
Let's invert this logic and turn it into a acceptable_to_fail list. The logging images are the ones that are most problematic,